src/west/manifest.py

```diff
@@ -183,6 +189,18 @@ def _load(self, data, sections):
 
         manifest = data.get('manifest')
 
+        url = config.get('self', 'remote', fallback='origin')
+        name = posixpath.basename(urlparse(url).path)
+        revision = config.get(name, 'revision', fallback='master')
+        path = os.path.join(name)
+
+        self_tag = manifest.get('self')
+        if self_tag:
+            path = self_tag['path']
```

Comment:
This would need a change if path were made optional, of course.

Reply:
yep. but that's a minor thing in that case.

Reply:
updated according to schema change